spec/fixtures/files/image_file.json

```diff
@@ -1,18 +1,10 @@
 {
-  "file": {
-    "created_at": "2019-05-20T14:46:08.240Z",
-    "updated_at": "2019-05-20T14:51:03.796Z",
+  "files": [{
     "file_name": "image_thumbnail_300.jpg",
     "file_type": "ImageThumbnail300",
     "content_type": "image/jpeg",
-    "byte_size": 8977,
```

Comment:
I'm not clear on why we're removing `byte_size` and `checksum` from the file fixture JSON? This is related to the comment about the `check_file_fixity` method in `Filestreams::Attacher`, obviously.